shallow_backup/git_wrapper.py

```diff
@@ -92,6 +93,11 @@ def move_git_repo(source_path, new_path):
 	"""
 	Moves git folder and .gitignore to the new backup directory.
 	"""
+	if os.path.exists(os.path.join(new_path, '.git')) or os.path.exists(os.path.join(new_path, '.gitignore')):
+		print_red_bold("Git repository already exists new path ({})".format(new_path))
```

Comment:
Change phrasing to:

```suggestion
		print_red_bold("A git repo already exists here: {}".format(new_path))
```

And also, the path should not be printed in bold (this is something I've been meaning to add to the `printing` utilities and haven't gotten around to yet.)

For each color, there should be a method `print_color_bold_path` that prints a `message` in bold and the path is the last item, and it prints that in non-bold style. Opening a separate issue for this.